systemtest/pom.xml

```diff
@@ -42,6 +42,12 @@
             <artifactId>hamcrest-all</artifactId>
             <version>${hamcrest.version}</version>
         </dependency>
+        <dependency>
+            <groupId>com.jayway.jsonpath</groupId>
+            <artifactId>json-path-assert</artifactId>
+            <version>2.2.0</version>
```

Comment:
Can you create a property with the version of this dependency in <properties> block of this POM file? I think it will be a good idea to keep versions for dependencies in the same place.